tools/stress-cluster/scripts/find_all_stress_packages.ps1

```diff
@@ -0,0 +1,47 @@
+param(
+    [string]$searchDirectory = '.',
+    [hashtable]$filters = @{}
+)
+
+class StressTestPackageInfo {
+    [string]$Namespace
+    [string]$Directory
+    [string]$ReleaseName
+}
+
+function findStressPackages([string]$directory, [hashtable]$filters = @{}) {
+    # Bare minimum filter for stress tests
+    $filters['stressTest'] = 'true'
+
+    Get-ChildItem -Recurse -Filter 'Chart.yaml' -PipelineVariable chartFile $directory 
+      | % { parseChart $chartFile }
+      | ? { matchesAnnotations $_ $filters }
+      | % { NewStressTestPackageInfo $_ $chartFile }
+}
+
+function parseChart([string]$chartFile) {
+    ConvertFrom-Yaml (Get-Content -Raw $chartFile)
```

Comment:
nit: generally I find adding a return keyword makes it a little more obvious what this does.